python/tchannel/tornado/peer.py

```diff
@@ -439,10 +484,17 @@ def send(self, arg1, arg2, arg3, traceflag=False, headers=None):
             Flag is for tracing.
         :param headers:
             Headers will be put int he message as protocol header.
+        :param attempt_times:
+            Number of times to attempts.
+        :param ttl:
+            Timeout for each request (ms).
+        :param retry_delay:
+            Delay between each retry (s).
```

Comment:
Some consistency between the units would be nice. Change `retry_delay` to `ms` and divide by 1000 before calling `sleep`.